crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -290,13 +296,14 @@ impl CheckpointExecutor {
         }
     }
 
+    #[instrument(level = "error", skip_all, fields(seq = ?checkpoint.sequence_number(), epoch = ?epoch_store.epoch()))]
     async fn schedule_checkpoint(
         &self,
         checkpoint: VerifiedCheckpoint,
         pending: &mut CheckpointExecutionBuffer,
         epoch_store: Arc<AuthorityPerEpochStore>,
     ) {
-        debug!("Executing checkpoint {:?}", checkpoint.sequence_number());
+        debug!("Scheduling checkpoint for execution");
```

Comment:
Why no checkpoint seq num here for context?

Reply:
I added the `#[instrument(level = "error", skip_all, fields(seq = ?checkpoint.sequence_number(), epoch = ?epoch_store.epoch()))]` on top of the function, which should add checkpoint seq num and epoch to the log message, e.g.
```
2023-04-14T03:23:48.805351Z DEBUG schedule_checkpoint{seq=1087014 epoch=755}: sui_core::checkpoints::checkpoint_executor: Scheduling checkpoint for execution
2023-04-14T03:23:48.805354Z DEBUG schedule_checkpoint{seq=1087014 epoch=755}: sui_core::checkpoints::checkpoint_executor: Preparing checkpoint for execution
2023-04-14T03:23:48.891360Z DEBUG schedule_checkpoint{seq=1087014 epoch=755}: sui_core::checkpoints::checkpoint_executor: Number of transactions in the checkpoint: 136
2023-04-14T03:23:49.315381Z DEBUG sui_core::checkpoints::checkpoint_executor: Checkpoint Execution Recv channel overflowed 179 messages
2023-04-14T03:23:49.315432Z DEBUG sui_core::checkpoints::checkpoint_executor: received checkpoint summary from state sync sequence_number=1210211
2023-04-14T03:23:49.315457Z DEBUG sui_core::checkpoints::checkpoint_executor: Bumping highest_executed_checkpoint watermark to 1086815
2023-04-14T03:23:49.315531Z DEBUG schedule_checkpoint{seq=1087015 epoch=755}: sui_core::checkpoints::checkpoint_executor: Scheduling checkpoint for execution
2023-04-14T03:23:49.315534Z DEBUG schedule_checkpoint{seq=1087015 epoch=755}: sui_core::checkpoints::checkpoint_executor: Preparing checkpoint for execution
2023-04-14T03:23:49.372200Z DEBUG schedule_checkpoint{seq=1087015 epoch=755}: sui_core::checkpoints::checkpoint_executor: Number of transactions in the checkpoint: 128
2023-04-14T03:23:49.759424Z DEBUG sui_core::checkpoints::checkpoint_executor: Checkpoint Execution Recv channel overflowed 101 messages
2023-04-14T03:23:49.759476Z DEBUG sui_core::checkpoints::checkpoint_executor: received checkpoint summary from state sync sequence_number=1210313
2023-04-14T03:23:49.759498Z DEBUG sui_core::checkpoints::checkpoint_executor: received checkpoint summary from state sync sequence_number=1210314
2023-04-14T03:23:49.759517Z DEBUG sui_core::checkpoints::checkpoint_executor: received checkpoint summary from state sync sequence_number=1210315
2023-04-14T03:23:49.759540Z DEBUG sui_core::checkpoints::checkpoint_executor: Bumping highest_executed_checkpoint watermark to 1086816
```